data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2471 @@
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+		"submunition" "spin4 print right"
+		"submunition" "spin4 print left"
+
+outfit "track3 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+		"submunition" "spin4 print right"
+		"submunition" "spin4 print left"
+
+outfit "spin4 print right"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"velocity" -25
+		"random velocity" 20
+		"inaccuracy" 5
+		"acceleration" -1
+		"live effect" "asteroid circle trail" 5
+		"drag" .1
+		"turn" -5
+		"lifetime" 40
+		"submunition" "track4 print right"
+
+outfit "spin4 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"velocity" -25
+		"random velocity" 20
+		"inaccuracy" 5
+		"acceleration" -1
+		"live effect" "asteroid circle trail" 5
+		"drag" .1
+		"turn" 5
+		"lifetime" 40
+		"submunition" "track4 print left"
+
+outfit "track4 print right"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+
+outfit "track4 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+
+outfit "asteroid laser 1"
+	weapon
+		sprite "projectile/asteroid laser"
+			"frame rate" 60
+			"random start frame"
+		sound "zapper"
+		"hit effect" "skylance impact"
+		"target effect" "asteroid target"
+		"inaccuracy" .3
+		"velocity" 200
+		"lifetime" 1
+		"reload" 1
+		"discharge damage" 1.7
+		"hull damage" 5.9
+
+effect "asteroid hit"
+	sprite "effect/point defense hit"
+		"frame rate" 13
+		"no repeat"
+	lifetime 1
+	"velocity scale" 0
+
+effect "asteroid target"
+	sprite "effect/finisher sparks"
+		"random start frame"
+		"frame rate" 20
+	"lifetime" 15
+	"random angle" 180
+
+effect "asteroid circle trail"
+	sprite "effect/asteroid circle trail"
+		"no repeat"
+		"frame time" 4
+	"lifetime" 20
+	"random velocity" 5
+	"random spin" 10
+	"velocity scale" 0.3
+
+outfit "Asteroid Weapon Large 3"
```

Comment:
This fires a total of 42 copies of the "asteroid missile". That's 1050 hull damage per shot, so a hull DPS of about 600? That's not unreasonable in itself (it's not reasonable for a weapon of only 16 tons, but the adapter should cover that).

The characteristics of the asteroid missile itself seem unreasonable, though - it's far too fast and too long-ranged.

Reply:
Reduced lifetime from 275 to 155 and acceleration from 2 to 1, while increasing turn from 20 to 60 to make it harder for them to miss